ev3dev2/motor.py

```diff
@@ -1804,26 +1758,36 @@ def on_for_degrees(self, left_speed, right_speed, degrees, brake=True, block=Tru
         ``degrees`` while the motor on the inside will have its requested
         distance calculated according to the expected turn.
         """
+
+        if left_speed == 0 and right_speed == 0 and degrees != 0:
+            raise ValueError("Can't travel a nonzero distance at zero speed")
```

Comment:
Two points: 
* There is probably some larger than zero speed threshold where motor will stall, so zero is not that special
* I can imagine a situation where speed is set by some function (e.g. based on user input), where it may legitimately cross zero. Not sure if raising an exception would be a good idea in this case.

Reply:
I went back-and-forth on this so I'm not attached to this behavior. I agree with your points; what would you suggest the behavior be?

If we ask the motor driver to go a nonzero distance with zero speed, it will report being `running stalled` indefinetely, and in turn will immediately cause our `wait_until_not_moving` to return. So would your suggestion be to adopt that behavior? That feels weird to me, but it's consistent with our single-motor methods (#515) so probably makes sense.

Reply:
> If we ask the motor driver to go a nonzero distance with zero speed, it will report being running stalled indefinetely, and in turn will immediately cause our wait_until_not_moving to return.

I think a return from `wait_until_not_moving()` here is an expected outcome. The speed the user asked was too small, so the motor is stalled, and is not moving.

Reply:
OK, thanks for the second opinion; I've updated the PR.